packages/tldraw/src/lib/shapes/geo/GeoShapeUtil.tsx

```diff
@@ -813,10 +814,10 @@ export class GeoShapeUtil extends BaseBoxShapeUtil<TLGeoShape> {
 
 	override onResize: TLOnResizeHandler<TLGeoShape> = (
 		shape,
-		{ initialBounds, handle, newPoint, scaleX, scaleY }
+		{ handle, newPoint, scaleX, scaleY, initialShape }
 	) => {
-		let w = initialBounds.width * scaleX
-		let h = initialBounds.height * scaleY
+		let w = initialShape.props.w * scaleX
+		let h = (initialShape.props.h + initialShape.props.growY) * scaleY
```

Comment:
cloud shape bounds rarely fits exactly inside the given width/height, so using the bounds here was causing a janky jump when resizing cloud shapes. Not sure how this wasn't an issue before 🤔